src/client/stories/index.js

```diff
@@ -3,10 +3,17 @@ import { storiesOf } from "@storybook/react";
 import { withKnobs } from "@storybook/addon-knobs";
 import { MapStory } from "./Map";
 import { WithTextStory, WithEmojiStory } from "./Buttons";
+import { VesselsStory } from "./Vessels";
 
 storiesOf("Button", module)
   .add("with text", WithTextStory)
   .add("with emoji", WithEmojiStory);
 
 const MapStories = storiesOf("Map", module).add("MapComponent", MapStory);
 MapStories.addDecorator(withKnobs);
+
+const VesselsStories = storiesOf("Vessels", module).add(
+  "VesselsComponent",
+  VesselsStory
```

Comment:
You need to have the components separately in Storybook:
 - VesselTeaser
 - VesselList

Reply:
Created two separate storybook
VesselTeaser
VesselList